tests/testthat/test_minDistToCells.R

```diff
@@ -0,0 +1,149 @@
+test_that("minDistToCells works",{
+  data("pancreasSCE")
+  
+  # works when cell types present and with negative distances returned
+  expect_silent(cur_sce <- minDistToCells(object = pancreasSCE,
+                 x_cells = pancreasSCE$CellType == "celltype_B",
+                 coords = c("Pos_X","Pos_Y"),
+                 img_id = "ImageName"))
+  
+  expect_true(is(cur_sce, "SingleCellExperiment"))
+  expect_s4_class(cur_sce , class = "SingleCellExperiment")
+  expect_true("distToCells" %in% names(colData(cur_sce)))
+  expect_true(class(cur_sce$distToCells) == "numeric")
+  expect_true(min(cur_sce$distToCells) < 0)
+  
+  # works on cell types when present and no negative distances returned
+  expect_silent(cur_sce_2 <- minDistToCells(object = pancreasSCE,
+                                          x_cells = pancreasSCE$CellType == "celltype_B",
+                                          coords = c("Pos_X","Pos_Y"),
+                                          img_id = "ImageName",
+                                          return_neg = FALSE))
+  
+  expect_true(is(cur_sce_2, "SingleCellExperiment"))
+  expect_s4_class(cur_sce_2 , class = "SingleCellExperiment")
+  expect_true("distToCells" %in% names(colData(cur_sce_2)))
+  expect_true(class(cur_sce_2$distToCells) == "numeric")
+  expect_true(min(cur_sce_2$distToCells) == 0)
+  
+  expect_equal(cur_sce[,cur_sce$distToCells > 0]$distToCells,cur_sce_2[,cur_sce_2$distToCells > 0]$distToCells)
+  
+  expect_equal(length(cur_sce[,cur_sce$distToCells < 0]),length(cur_sce_2[,cur_sce_2$distToCells == 0]))
+
+  # works on cell types when not present in some image and with negative distances returned
+  expect_silent(cur_sce_3 <- minDistToCells(object = pancreasSCE,
+                                          x_cells = pancreasSCE$CellType == "celltype_A",
+                                          coords = c("Pos_X","Pos_Y"),
+                                          img_id = "ImageName"))
+  
+  expect_true(is(cur_sce_3, "SingleCellExperiment"))
+  expect_s4_class(cur_sce_3 , class = "SingleCellExperiment")
+  expect_true("distToCells" %in% names(colData(cur_sce_3)))
+  expect_true(class(cur_sce_3$distToCells) == "numeric")
```

Comment:
Here it's safer to test `is(cur_sce_3$distToCells, "numeric")`

Reply:
for some images in this test NAs were returned so the proposed test fails. I believe it is ok like this and added the proposed test to check if the whole images had NAs below.